quickstarts/iFrame.mdx

```diff
@@ -0,0 +1,297 @@
+
+| Channel | Content Style |
+| --- | --- |
+| Email | 1. Custom Code (HTML) with option to use custom variables via the handlebars , , syntax. |
+|  | 2. Click and place UI items with the visual workflow editor. |
+| SMS | Text with the option to use handlebars syntax, to inject custom variables. |
+| Chat | Text with the option to use handlebars syntax, to inject custom variables. |
+| In-App | Text |
+
+These are the steps to create a workflow:
+
+1. Click **Workflow** on the left sidebar of your Novu dashboard.
+2. Click the **Create Workflow** button on the top right.
+3. The name of the new workflow is currently **“Untitled”**. Rename it to a more suitable title.
+4. Select **In-App** as the channel you want to add.
+
+<img
+  style={{ borderRadius: '0.5rem' }}
+  src="/images/Quickstarts/iFrame-quickstart/1.png"
+/>
+
+Select the in-app channel
+
+1. Click on the recently added “In-App” channel and configure it according to your preferences. Once you’re done, click “Update” to save your configuration.
+
+<img
+  style={{ borderRadius: '0.5rem' }}
+  src="/images/Quickstarts/iFrame-quickstart/2.png"
+/>
+
+Configure the in-app node as per your need
+
+I’ll briefly explain the function of each label in the image above:
+
+- **1-Preview**: This shows you a glimpse of how each notification item will look in the Notification Center UI.
+- **2-Avatar**: If turned on, each notification item will show the avatar of the subscriber.
+- **3-Action**: With this, you can add a primary and secondary call to action button to each notification item.
+- **4-Notification Feeds**: This displays a stream of specific notifications. You can have multiple feeds to show specific notifications in multiple tabs.
+- **5-Redirect URL**: This is the URL to which a subscriber can be directed when they click on a notification item.
+- **6-Filter**: This feature allows you to configure the criteria for delivering notifications. For instance, you can apply a filter based on a subscriber’s online status to send them an email if they were online within the last hour. Read **[more about filters](https://docs.novu.co/platform/step-filter/#subscriber-seen--read-filters)**.
+- **7-Editor**: You can add text that you want to be displayed in each notification item. Additionally, you can specify custom variables using **`{{ }}`**. This means you can inject variables from your code into a notification item’s text via a payload.
+
+In our case, we’ll use the custom variables functionality, as shown below:
+
+<img
+  style={{ borderRadius: '0.5rem' }}
+  src="/images/Quickstarts/iFrame-quickstart/3.png"
+/>
+
+We're gonna use custom variables here
+
+Feel free to add only text for now and rename the workflow to quickstart. It automatically creates a slug-like Identifier that will be needed in later steps to trigger a notification.
+
+<img
+  style={{ borderRadius: '0.5rem' }}
+  src="/images/Quickstarts/iFrame-quickstart/4.png"
+/>
+
+Renaming the workflow to 'quickstart'
```

Comment:
Is this meant to be a header or a comment that refers to the image above?

Reply:
Comment that refers to the image above. 

Done.